components/external-api-framework/org.wso2.carbon.identity.external.api.token.handler/pom.xml

```
@@ -0,0 +1,200 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
```

```suggestion
  ~ Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
```
Check other places as well